packages/financial-templates-lib/src/clients/OptimisticOracleEventClient.js

```diff
@@ -146,11 +152,18 @@ class OptimisticOracleEventClient {
         timestamp: event.returnValues.timestamp,
         ancillaryData: event.returnValues.ancillaryData ? event.returnValues.ancillaryData : "0x",
         proposedPrice: event.returnValues.proposedPrice,
+        currency: requestData.currency,
```

Comment:
It's probably an oversight that we don't emit `currency` as part of these events, but fortunately we do store it on-chain. I'm open to also parallelizing the `_getRequestData` call but I think its possibly over-optimizing since this monitor isn't expected to be fetching that many events per run, in production. WDYT? @UMAprotocol/eng

Reply:
ye, it totally should be part of the event. the way you have it here is a good workaround for now.

Reply:
prehaps add a comment explaining this

Reply:
Yeah, there's a tradeoff because event data also drives up gas costs. I think your solution here is totally reasonable and agree on the comment suggestion.